lib/plug/router.ex

```diff
@@ -253,12 +253,35 @@ defmodule Plug.Router do
 
       @doc false
       def dispatch(%Plug.Conn{} = conn, opts) do
-        {_path, fun} = Map.fetch!(conn.private, :plug_route)
+        metadata = %{conn: conn}
+        start = System.monotonic_time()
+        {path, fun} = Map.fetch!(conn.private, :plug_route)
+
+        :telemetry.execute(
+          [:plug, :router_dispatch, :start],
+          %{path: path, system_time: System.system_time()},
```

Comment:
I think we call it route on Phoenix. Should we do the same here?

Reply:
Also, path should be in metadata. :)

Reply:
ah duh, hehe, yea, I've moved and renamed it.